public/css/classic.css

```diff
@@ -182,15 +186,101 @@ select.form-input {
     border: 1px solid #333333;
     color: #FBFBFB;
 }
+.website-nav .pagination p {
+	display: none;
+}
+.website-nav {
+	padding-bottom: 2px!important;
+}
+.website-nav .pagination a {
+	display: inline!important;
+}
+.website-nav #page-prev li{
+	float: left;
+}
+.website-nav #page-next li{
+	float: right;
+}
+
+#description-box code {
+	font-size: 1.3em;
+}
+
 .results.box {
     padding: 1px!important;
     border-top: none;
     border-radius: 0 0 4px 4px;
 }
-.box, #description-box, .comment-box, .comment-submit,input#show-filelist:checked~#filelist {
+.box, #description-box, .comment-box,input#show-filelist:checked~#filelist {
     background-color: #FBFBFB;
     border: 2px solid #666666;
 }
+
+.comment-box {
+	font-size: .9em;
+    border-radius: 5px;
+    margin: 0 0 4px 0;
+    padding: 4px;
+	min-height: 70px;
+}
+
+span.comment-index+p {
+	margin-top: 0;
+}
+
+span.comment-index+p a {
+	font-weight: bold;
+    padding-top: 2px;
+    font-size: .9em;
+	color: black;
+	text-decoration: none;
+    line-height: 22px;
+}
+span.comment-index+p a:hover {
+	text-decoration: underline;
+}
+
+.comment-index {
+	margin: 3px 0 0 0;
+}
+
+#solution {
+    margin-top: -3px;
+}
+
+.comment-index a::before {
+	content: '';
+}
+.comment-index a {
+	font-weight: bold;
+	color: black;
+    float: right;
+    margin-left: 9px;
+}
+.comment-index a:hover {
+	color: #44A0A0;
+}
+
+
+.comment-submit {
+	border: none;
+	background: none;
+    margin-top: 15px!important;
+}
+.comment-submit .comment-form {
+	margin: 0;
+}
+.comment-form h3 {
+	margin-top: 20px;
+}
+
+.comment-submit textarea{
+    height: auto!important;
+    width: auto!important;
```

Comment:
isn't `!important` suppose to have whitespace in front?

Reply:
no